scalastyle-config.xml

```diff
@@ -1,117 +0,0 @@
-<scalastyle>
```

Comment:
It is outdated; If we'd like to use smth like that I think we need to look towards scalafmt.